cnd/diesel.toml

```diff
@@ -0,0 +1,5 @@
+# For documentation on how to configure this file,
+# see diesel.rs/guides/configuring-diesel-cli
```

Comment:
We are not using the `diesel-cli`. Do we still need this file?

Reply:
You may be right, since we run the migrations in code i don't think we need this.  Will remove and test.